### ISSUE TITLE
MqttOwfs as a service

### ISSUE THREAD

**Issue by TomDeVlaminck** [FragJage/MqttOwfs#11]
Hello, first of I want to thank you for writing this service.
I have modified it so that I can transfer all the counter values and temp values to mqtt and than I visualize it with grafana.

My problem is that i want to start it as a service.

So I added a file /etc/systemd/system/multi-user.target.wants/mqttowfs.service 
But this doesnt start the mqttowfs process on boot.
Do you have an example service file? That would help me a lot.
Maybe add it to your already fantastic wiki.
Thank you very much
Tom

[Unit]
Description=MqttOwfs 
After=network-online.target mosquitto.service
Wants=network-online.target mosquitto.service

[Service]
Type=forking
User=daemon
Group=daemon
ExecStart=/usr/bin/MqttOwfs -start --configfile /etc/MqttOwfs.conf
Restart=always

[Install]
WantedBy=multi-user.target

**Comment by FragJage** [comment]
Hello, and thank you.
I don't use systemd but init.d to start my service.
I will complete the wiki with an example use init.d next weekend.
3 questions to identify the problem : 
 - Do you have enable the service `systemctl enable <nom du service>.service`
 - Can you start the service `systemctl start <nom du service>.service`
 - Does the user daemon have sufficient privileges to start the service ?
Best regard
Frag

**Comment by FragJage** [comment]
Wiki update to explain how start daemon with init.d.

**Comment by TomDeVlaminck** [comment]
Hello, 
Thank you for adding the instructions. Sorry that it has taken so long for me to respond, but it is quite busy.
Before I had systemd configured, but I removed this configuration and installed it by your instructions.
It now runs as a service (I figured out with systemd how to run it also) - I will see how long it will run because with the systemd it only ran 3 or 4 minutes.
Now it is running more than 15 minutes so I guess it is OK.
I will pose a feature request to add the A and B counters and other values to your code.
Greetings & Happy new year & a good health!

<img width="286" alt="Screenshot 2020-12-31 at 00 34 16" src="https://user-images.githubusercontent.com/6303627/103386720-8a75ef00-4b00-11eb-9966-189ec3449197.png">
<img width="285" alt="Screenshot 2020-12-31 at 00 32 09" src="https://user-images.githubusercontent.com/6303627/103386722-8b0e8580-4b00-11eb-8b9c-3d7fd96c53d9.png">


Tom
